### PR TITLE
update paper egg

### PR DESCRIPTION
**Pull request by parkervcp**
Updates the paper egg for 1.17 support with java 11 by default

### CODE REVIEW

**Comment by TekExplorer** [comment]
you removed the [eula] feature :eyes:

**Comment by TekExplorer** [comment]
also just gonna point out for record's sake that this also updates the egg to use paper's v2 api

File: database/seeds/eggs/minecraft/egg-paper.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
         "version": "PTDL_v1"
     },
-    "exported_at": "2020-11-03T04:22:56+00:00",
+    "exported_at": "2020-12-04T12:08:20-05:00",
     "name": "Paper",
     "author": "parker@pterodactyl.io",
     "description": "High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.",
-    "features": ["eula"],
-    "image": "quay.io\/pterodactyl\/core:java",
+    "features": null,
```
Comment (DaneEveritt) [inline database/seeds/eggs/minecraft/egg-paper.json]: This should not have been removed?

File: database/seeds/eggs/minecraft/egg-paper.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
+    "image": "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-11",
```
Comment (DaneEveritt) [inline database/seeds/eggs/minecraft/egg-paper.json]: Unrelated, can we merge these images into the core repo?

**Review by DaneEveritt — CHANGES_REQUESTED** [review]
see comments

File: database/seeds/eggs/minecraft/egg-paper.json (hunk 2 above)
Comment (parkervcp) [reply database/seeds/eggs/minecraft/egg-paper.json]: I can pr a java-11 image up to the core images repo

**Comment by DaneEveritt** [comment]
@parkervcp can you fix that features issue indicated above and switch the image to the core java 11 image now?

**Comment by parkervcp** [comment]
@DaneEveritt as soon as I'm at a place where I can I will.

**Comment by DaneEveritt** [comment]
Went ahead and fixed it for you @parkervcp :)

**Pull request merged**